src/main/webapp/resources/js/pages/admin/components/user/CreateNewUserForm.jsx

```diff
@@ -0,0 +1,284 @@
+          },
+        ]}
+      >
+        <Select>
+          {systemRoles.map((role, index) => (
+            <Select.Option
+              key={`create-new-user-account-role-${index}`}
+              value={role.code}
+            >
+              {role.name}
+            </Select.Option>
+          ))}
+        </Select>
+      </Form.Item>
+      {!emailConfigured && (
+        <Alert
+          style={{ marginBottom: SPACE_SM }}
+          message={i18n("CreateNewUserForm.emailConfigured.alert.title")}
+          description={
+            <Typography.Paragraph>
+              {i18n("CreateNewUserForm.emailConfigured.alert.description")}
+            </Typography.Paragraph>
+          }
+          type="warning"
+          showIcon
+        />
+      )}
+      <Form.Item name="activate" valuePropName="checked">
+        <Checkbox
+          disabled={!emailConfigured}
+          checked={activationEmail}
+          onChange={(e) => setActivationEmail(e.target.checked)}
+        >
+          {i18n("CreateNewUserForm.form.activate.label")}
+        </Checkbox>
+      </Form.Item>
+      {!activationEmail && (
+        <>
+          <Alert
+            style={{ marginBottom: SPACE_SM }}
+            message={i18n("CreateNewUserForm.changePassword.alert.title")}
+            description={
+              <Typography.Paragraph>
+                <List
+                  header={i18n(
+                    "CreateNewUserForm.changePassword.alert.description"
+                  )}
+                  dataSource={passwordRules}
+                  renderItem={(item) => <List.Item>{item}</List.Item>}
+                />
+              </Typography.Paragraph>
+            }
+            type="info"
+            showIcon
+          />
+          <Form.Item
+            label={i18n("CreateNewUserForm.form.label.password")}
+            name="password"
+            rules={[
+              {
+                required: true,
+                message: i18n("CreateNewUserForm.changePassword.alert.rule1"),
+              },
+              {
+                min: 8,
+                message: i18n("CreateNewUserForm.changePassword.alert.rule2"),
+              },
+              {
+                pattern: new RegExp("^.*[A-Z].*$"),
+                message: i18n("CreateNewUserForm.changePassword.alert.rule3"),
+              },
+              {
+                pattern: new RegExp("^.*[a-z].*$"),
+                message: i18n("CreateNewUserForm.changePassword.alert.rule4"),
+              },
+              {
+                pattern: new RegExp("^.*[0-9].*$"),
+                message: i18n("CreateNewUserForm.changePassword.alert.rule5"),
+              },
+              {
+                pattern: new RegExp("^.*[^A-Za-z0-9].*$"),
+                message: i18n("CreateNewUserForm.changePassword.alert.rule6"),
+              },
+            ]}
+          >
+            <Input.Password />
+          </Form.Item>
+        </>
+      )}
+      <Form.Item>
```

Comment:
Can you use the modal buttons to submit this form.

Reply:
Please see 43fb8636af8f5ca7b3066e6829a53e237380cd83.